patches/tModLoader/Terraria/ModLoader/Default/UnloadedGlobalItem.cs

```diff
@@ -9,6 +9,11 @@ public class UnloadedGlobalItem : GlobalItem
 	[CloneByReference] // safe to share between clones, because it cannot be changed after creation/load
 	internal IList<TagCompound> data = new List<TagCompound>();
 
+	[CloneByReference]
+	internal string modPrefixMod = null;
+	[CloneByReference]
+	internal string modPrefixName = null;
```

Comment:
```suggestion
	[field: CloneByReference]
	public string ModPrefixMod { get; internal set; } = null;
	[field: CloneByReference]
	public string ModPrefixName { get; internal set; } = null;
```
To follow the same design as UnloadedItem.ModName/ItemName. There are use cases for this to be accessible by other mods (including one of my mods). Obviously the case change should be applied to all places where it's used.

Reply:
Done.